Frameworks/CoreText/CTTypesetter.mm

```diff
@@ -84,6 +84,7 @@ static float fractionPixels(float pixels) {
 
 static CFIndex _DoWrap(CTTypesetterRef ts, CFRange range, WidthFinderFunc widthFunc, void* widthParam, double offset, CTLineRef* outLine) {
```

Comment:
Why do we still need this method? Presumably for Draw until we move it to DWrite?